### ISSUE TITLE
Checkbox common component - Text labels not picked up by screen reader on Android 

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16484]
## What Happened?
On Android, the text labels for the checkbox are not being read by the screen reader. TalkBack is only picking up the a11yHints. 

Affects the checkboxes on these pages:
- Edit Address
- Edit Direct Deposit Information

## Specs:
- Device: Android - Google Pixel 3XL with TalkBack enabled

## Steps to Reproduce
1. With TalkBack enabled on Android, navigate to either the edit address or edit direct deposit pages
2. Select the checkbox element and note that the text label is not read

## Desired Behavior
- The checkbox text labels should be read by the screen reader (TalkBack)

## Acceptance Criteria
- [ ] The checkbox text labels will be read by the screen reader (TalkBack)

## Bug Severity
- 4 - Low

## Linked Story (s)
- Story #15442

## Screen Shot (s) and Additional Information

**Comment by esther-k** [comment]
**11/30**
Pass on Android (bug only affects Android). Text labels are not being picked up by the screen reader
